### PR TITLE
Migration auth UI

### PR DESCRIPTION
**Pull request by bhatiadheeraj**

### CODE REVIEW

File: ui/src/settings.vue [hunk 1]
```diff
@@ -731,6 +826,15 @@ h5 {
     margin-bottom: 20px;
     opacity: 0.7;
 }
+.well{
```
Comment (soichih) [inline ui/src/settings.vue:829]: Please add space between well and {

File: ui/images/account.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
+<div class="fixed-top">
```
Comment (soichih) [inline ui/images/account.html:1]: What is this file for?

File: ui/src/settings.vue [hunk 3]
```diff
@@ -18,7 +18,7 @@
         <b-container v-if="ready">
 
             <!--profile-->
-            <div v-if="tab == 0">
+            <div v-if="tabs[tab].id == 'profile'">
```
Comment (soichih) [inline ui/src/settings.vue:21]: Do you think we should create a simple computed property like this?

```
computed: {
tabID() {
     return this.tabs[tab].id; 
}
```

And use this in the template?

```
<div v-if="tabID == 'profile'">
```

Maybe it's silly?

File: ui/src/settings.vue [hunk 4]
```diff
@@ -500,25 +574,22 @@ export default {
             this.fullname = res.data.fullname;
             if(res.data.profile) lib.mergeDeep(this.profile, res.data.profile);
             this.ready = true;
-
-            //this.tab = won't work unless it's next tick from mounted()
+            const jwt = localStorage.getItem("jwt");
```
Comment (soichih) [inline ui/src/settings.vue:577]: You can use `Vue.config.jwt` to access the jwt. You don't need to do this.

**Review by soichih — CHANGES_REQUESTED** [review]
I've reviewed some changes.

Can you remove all those ui/images/*.png ? I don't think you need them, right?

**Comment by bhatiadheeraj** [comment]
> I've reviewed some changes.
> 
> Can you remove all those ui/images/*.png ? I don't think you need them, right?

In which folder should I store the cilogon.png for OpenID connect

File: ui/src/settings.vue [hunk 5]
```diff
@@ -674,7 +745,24 @@ export default {
                 members : [],
                 admins : [],
             }
-        }
+        },
+        connect(type) {
+            window.location = "api/auth/"+type+"/associate/"+this.jwt;
```
Comment (soichih) [inline ui/src/settings.vue:750]: The URL is wrong. This needs to be

```
window.location = Vue.config.auth_api+"/"+type+"/associate/"+this.jwt;
```

**Review by soichih — CHANGES_REQUESTED** [review]

**Pull request merged**